config/standard/test/tasks.spec.js

```diff
@@ -535,11 +535,11 @@ describe('Standard Configuration Tasks', () => {
         } else {
           it('should not have "immunization-missing-visit" visit task', () => {
             // given
-            harness.pushMockedReport(
+            harness.pushMockedDoc(
```

Comment:
```
pushMockedDoc: report without subject id (patient_id, patient_uuid, place_id, etc). Setting default to "default_subject".
pushMockedDoc: report without subject id (patient_id, patient_uuid, place_id, etc). Setting default to "default_subject".
pushMockedDoc: report without subject id (patient_id, patient_uuid, place_id, etc). Setting default to "default_subject".
pushMockedDoc: report without subject id (patient_id, patient_uuid, place_id, etc). Setting default to "default_subject".
```

In build logs. Not sure if it's an issue

Reply:
I'm not sure what the hiearachy is for this project, but you might need a harness.default.json file with this breaking change.